vispy/gloo/context.py

```diff
@@ -166,7 +166,12 @@ def flush_commands(self, event=None):
         """
         if self._do_CURRENT_command:
             self._do_CURRENT_command = False
-            self.shared.parser.parse([('CURRENT', 0)])
+            canvas = get_current_canvas()
+            if canvas and hasattr(canvas, '_backend'):
+                fbo = canvas._backend._vispy_get_fb_bind_location()
+            else:
+                fbo = 0
+            self.shared.parser.parse([('CURRENT', 0, fbo)])
```

Comment:
yeah I'm not 100% sure how this stuff works, we might have to wait until @almarklein pops up again to get some insight if you can't get it from the code